packages/storybook/src/schematics/configuration/root-files/.storybook/config.js

```diff
@@ -2,10 +2,12 @@ import { configure, addDecorator } from '@storybook/angular';
 import { withKnobs } from '@storybook/addon-knobs';
 
 function loadStories() {
```

Comment:
Is this for the global storybook? or is this for each individual to extend, If the answer is for individuals you should have this functionality in here

Reply:
This is for running a global storybook that will pick up all stories in the monorepo

Reply:
What about extending the base config for the individual one? Wouldn't a better option to add a new application in angular.json for the global storybook to run?

Reply:
Adding a separate app would work, but the benefit of this `.storybook` folder configuration is that people who are used to running `start-storybook` from the command line will get the global storybook